doc/source/examples/multiphysics/dam-break-VOF/dam-break-VOF.rst

```diff
@@ -0,0 +1,255 @@
+
+`Sharpening frequency` is an integer parameter that defines the 
+frequency of interface sharpening; sharpening threshold defines 
+a phase fraction threshold for interface sharpening (generally :math:`0.5`);
+and interface sharpness is a model parameter which is generally in
+the range of :math:`(1-2]`.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Interphase sharpening
+    #---------------------------------------------------
+    subsection interface sharpening
+        set sharpening threshold        = 0.5
+        set interface sharpness         = 2
+        set sharpening frequency      	= 10
+    end
+
+In the ``initial condition``, the initial velocity and initial position 
+of the liquid phase are defined. The liquid phase is initially 
+defined as rectangle of length :math:`= 3.5` and height ::math:`= 7`.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Initial condition
+    #---------------------------------------------------
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 0; 0; 0
+        end
+        subsection VOF
+             set Function expression = if (x<3.5 & y<7 , 1, 0)
+        end
+    end
+
+The ``source term`` subsection defines the gravitational acceleration:
+
+.. code-block:: text
+    
+    #---------------------------------------------------
+    # Source term
+    #---------------------------------------------------
+    subsection source term
+        set enable = true
+        subsection xyz
+            set Function expression = 0;-1.0; 0
+        end
+    end
+
+Two fluids are present in this simulation, hence in the ``physical 
+properties`` subsection, their physical properties should be defined:
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Physical Properties
+    #---------------------------------------------------
+    subsection physical properties
+        set number of fluids     = 2
+        subsection fluid 0
+            set density              = 0.02
+            set kinematic viscosity  = 0.1
+        end
+        subsection fluid 1
+            set density              = 0.9982
+            set kinematic viscosity  = 0.01
+        end
+    end
+
+In the ``mesh adaptation subsection``, adaptive mesh refinement is 
+defined for ``velocity``. ``min refinement level`` and ``max refinement 
+level`` are 4 and 5, respectively.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Mesh Adaptation
+    #---------------------------------------------------
+    subsection mesh adaptation
+        set type                    = kelly
+        set variable                = velocity
+        set fraction type           = fraction
+        set max refinement level    = 5
```

Comment:
we are missing the `mesh` subsection of the .prm to understand that (initial refinement = 4, so you could put max refinement level to 6 and min level to 3 I think)

Reply:
still to be addressed ;)

Reply:
Attention to detaal